roles/core/dhcp_server/vars/RedHat_9.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
If `RedHat_9.yml` file is similar to `RedHat_8.yml`, you could rename `RedHat_8.yml -> RedHat.yml` instead of duplicating the file.